src/components/Reports/GDPRPermissions/SubscriberGdprPermissions.js

```diff
@@ -116,8 +87,8 @@ const SubscriberGdprPermissions = ({
                       <FormattedMessage id="subscriber_gdpr.empty_data" />
                     </span>
                   </td>
-                  <td></td>
-                  <td></td>
+                  <td />
+                  <td />
```

Comment:
Is it legal React code?

Reply:
Every React component can be self-closing: <td />. <td></td> is also an equivalent.